modules/optimization/test/src/optimizationreporters/QuadraticMinimize.C

```diff
@@ -8,6 +8,7 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "QuadraticMinimize.h"
+#include "libmesh/petsc_vector.h"
```

Comment:
```suggestion
#include "libmesh/petsc_vector.h"
```
why now

Reply:
It was getting the petscvector from a class it was inheriting from but in that class I changed it to a forward declare.